src/windows/InAppBrowserProxy.js

```diff
@@ -176,6 +179,38 @@ var IAB = {
 
                 browserWrap.appendChild(popup);
 
+                var closeHandler = function (e) {
+                    setTimeout(function () {
+                        IAB.close(navigationEventsCallback);
+                    }, 0);
+                };
+
+                if (features.indexOf("hardwareback=yes") > -1 || features.indexOf("hardwareback") === -1) {
+                    hardwareBackCallback = function () {
+                        if (browserWrap.style.display === 'none') {
+                            // NOTE: On Windows Phone 8.1 backbutton handlers have to throw an exception in order to exit the app
+                            throw 'Exit the app';
```

Comment:
From what i understand, this doesn't necessarily mean that app will exit after this - this just means that app will fall back to default system navigation behavior. If that's true please update comment above to reflect this.